tools/package.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+mkdir -p pkg
+zip pkg/logdna-s3.zip -r node_modules/ index.js package.json lib/*.js
```

Comment:
I'd probably suggest moving the `npm ci --production` in here, rather than run it from the Jenkinsfile

Reply:
Done! Added!